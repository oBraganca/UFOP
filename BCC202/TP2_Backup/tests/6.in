7
4 0
5 3
4 4
5 0
4 3
0 0
1 3
0