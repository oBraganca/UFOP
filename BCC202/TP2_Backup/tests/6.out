Test 1:
YES
13|36|65|50|00|04|45|52|22
